### DISCUSSION TITLE
Auto add mailbox based on selected custom domains

### DISCUSSION THREAD

**Discussion by 0nl1n1n** [simple-login/app#2286] [Feature Requests]
When I create an alias such as alias32@domain1.com I already know which mailbox I will associate, but still have to unselect the default one, and then select the one I want.

To streamline this process, it would be great, probably at the custom domain creation, to automatically route all the aliases by default to a particular email.

**Comment by ZelionGG** [comment]
Deffo would love this, having to uncheck the default mailbox every single time to select the one I want when I generate my aliases from BitWarden is quite a heavy process, I often forget to do the change and then receive my verification emails on the wrong address...

**Comment by eopo** [comment]
The current version (Docker image app-ci) does include that feature. Custom Domains can be assigned a default mailbox for auto-created aliasses, also manually creating an alias does show a dropdown to select the mailbox.
